conf/modules.config

```diff
@@ -50,6 +66,7 @@ process {
     // Otherwise the .dict includes paths from Nextflow's work/ directory
     withName: SAMTOOLS_DICT {
         ext.args = { "--uri ${task.publishDir.path[0]}/${fasta.name}.gz" }
+        publishDir: [ enabled: false ]
```

Comment:
This feels redundant with the `|| filename.endsWith('.dict') ? null : filename` above. Do we really need both ?

Reply:
just tried and you are right